src/node.cc

```diff
@@ -515,6 +534,18 @@ inline void PlatformInit() {
   RegisterSignalHandler(SIGINT, SignalExit, true);
   RegisterSignalHandler(SIGTERM, SignalExit, true);
 
+  {
+    struct sigaction sa;
+    memset(&sa, 0, sizeof(sa));
+    sa.sa_sigaction = TrapWebAssemblyOrContinue;
+#ifndef __FreeBSD__
+    sa.sa_flags = SA_RESETHAND;
```

Comment:
This restores the dispostion to `SIG_DFL` after the first SIGSEGV. I don't think that's what you want. :-)